src/linker/Linker.Steps/XmlCustomAttributesStep.cs

```diff
@@ -5,35 +5,34 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Runtime.InteropServices;
 using System.Text;
 using System.Text.RegularExpressions;
 using System.Xml.XPath;
 using Mono.Cecil;
 
-namespace Mono.Linker
+namespace Mono.Linker.Steps
 {
-	class XmlCustomAttributeSource
+	class XmlCustomAttributesStep : BaseStep
```

Comment:
Consider renaming this to LinkAttributesStep or something like that - to match the "feature name", the "XML" part is effectively an implementation detail.